### ISSUE TITLE
Member Management > Cannot load the member list in the channel that has aout 99 members

### ISSUE THREAD

**Issue by tlle14** [linagora/Twake-Mobile#874]
# Precondition
Production
The channel has about 99 members

# Actual Result
When opening Member Management, the member list cannot be loaded
https://user-images.githubusercontent.com/69950655/131646322-58a9000d-532a-4c5a-8c8f-db90ace90ed6.mp4

# Expected Result
The member list should be loaded anyway.

# Environment
- Build version: 3.0.6
- Device:
Samsung Galaxy A51 - Android 11 - 6.5 inch
- Production

**Comment by bobs4462** [comment]
Can you write the name of the channel where it can reproduced? May be we cannot render bot members...

**Comment by tlle14** [comment]
> Can you write the name of the channel where it can reproduced? May be we cannot render bot members...

Please visit Home Workspace > General channel on Production

**Comment by bobs4462** [comment]
```https://beta.twake.app/internal/services/channels/v1/companies/56393af2-e5fe-11e9-b894-0242ac120004/workspaces/42b997e8-e15b-11ea-bf1d-0242ac120004/channels/61bb4764-6d1e-41eb-a83c-0242ac120004/members```
@RomaricMourgues I'm getting 500 for any channel on production, QA works fine

**Comment by tlle14** [comment]
Hello @bobs4462 I don't think this issue is fixed. But it's from backend, am I correct?

**Comment by bobs4462** [comment]
I couldn't reproduce it

**Comment by tlle14** [comment]
Have you tried to open the channel "General" on Linagora Workspace on Production?

**Comment by bobs4462** [comment]
That's right
@RomaricMourgues 
```
https://beta.twake.app/internal/services/channels/v1/companies/56393af2-e5fe-11e9-b894-0242ac120004/workspaces/42b997e8-e15b-11ea-bf1d-0242ac120004/channels/61bb4764-6d1e-41eb-a83c-0242ac120004/members
```
this request fails with 500 for some reason

**Comment by bobs4462** [comment]
@RomaricMourgues any ideas regarding this one?

**Comment by RomaricMourgues** [comment]
I can't get the 500 error from my side, could you detail the requests parameters ? (I did a simple GET without any parameters).
For your information, any request with limit > 100 will fail.

**Comment by bobs4462** [comment]
Yes the limit is the issue, as it's set > 100, but it fails only for some channels

**Comment by RomaricMourgues** [comment]
Yes, only with channels having more than 100 members :)
Could you set the limit to less than 100 ?

**Comment by bobs4462** [comment]
Yes, already did :), I implemented it with pagination, so it might take a few sequential requests to get a full a list

**Comment by RomaricMourgues** [comment]
This is the goal yes !
By the way the 500 error comes from a limitation in our database for IN queries. We'll do a fix someday for this.
